README.md

```diff
@@ -9,6 +9,18 @@ For application features, build instructions and user guides see the [srsRAN doc
 
 ## Usage
 
+### Build
+To build this snap, you will need a machine with the following requirements:
+- Processor: x86-64 dual-core processor
+- OS: Ubuntu20-04
+- Memory: 4GB RAM
+
+run
+```bash
+SNAPCRAFT_BUILD_ENVIRONMENT_MEMORY=4G snapcraft --destructive-mode
```

Comment:
No, in case of using multipass I think it is 2G.
But anyway in our case it doesn't really matter as we use the host env directly with --destructive-mode.
Removed :)